`WebviewTag` type is missing in `electron` module

Today I tried to upgrade Electron dependency from v14 to v16 and TypeScript compiler reported one error which seems a regression of Electron type definitions.

## Environment

- macOS 11.6
- Node v17.3
- Electron v16.0.7

## Repro

- test.ts

```typescript
import { WebviewTag } from 'electron';
console.log(WebviewTag);
```

- tsconfig.json

```json
{
  "compilerOptions": {
    "module": "commonjs",
    "moduleResolution": "node",
    "removeComments": true,
    "preserveConstEnums": true,
    "noImplicitAny": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noEmitOnError": true,
    "noFallthroughCasesInSwitch": true,
    "noPropertyAccessFromIndexSignature": true,
    "strictNullChecks": true,
    "strictBindCallApply": true,
    "strictPropertyInitialization": true,
    "target": "es2017",
    "sourceMap": true,
    "esModuleInterop": true,
    "resolveJsonModule": true
  },
  "include": [
    "*.ts"
  ]
}
```

And run `tsc -p .`

## Expected behavior

No compile error

## Actual behavior

Got the following error

```
test.ts:1:10 - error TS2305: Module '"electron"' has no exported member 'WebviewTag'.

1 import { WebviewTag } from 'electron';
           ~~~~~~~~~~
```

## Note

With Electron v14, this error did not happen.

I noticed three modules are defined in electron.d.ts in Electron v16; `electron`, `electron/main`, `electron/renderer`. So I tried the following but it also caused the same error.

```typescript
import { WebviewTag } from 'electron/renderer';
```

`WebviewTag` is defined in `Electron` namespace so it is accessible via `Electron.WebviewTag`, but it is not included in the above three modules.

--- a/src/primary-interfaces.ts
+++ b/src/primary-interfaces.ts
@@ -75,6 +75,11 @@ export const generatePrimaryInterfaces = (
         );
       }
     }
+    if (module.type === 'Element') {
+      newTypeAliases.push(
+        `type ${_.upperFirst(module.name)} = Electron.${_.upperFirst(module.name)};`,
+      );
+    }
     constDeclarations.push(...newConstDeclarations);
     if (module.process.main && module.process.renderer) {
       TargetNamespace = CommonNamespace;

